<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlg_l0filter_para</class>
 <widget class="QWidget" name="dlg_l0filter_para">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="1" column="0">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>34</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QCheckBox" name="norm_L0">
        <property name="text">
         <string>norm L0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="vert_L0">
        <property name="text">
         <string>vert L0</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="edge_recovery">
        <property name="text">
         <string>edge recovery</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="3">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>34</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>34</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>34</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QPushButton" name="L0filter">
     <property name="text">
      <string>L0filter</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>normKnn</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="normKnn"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>norm sparsity</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="norm_sparsity">
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>norm auxiliary</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="norm_auxiliary">
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>norm increment</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="norm_increment">
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>norm iteration</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="norm_iteration"/>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>vertKnn</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="vertKnn"/>
        </item>
       </layout>
      </item>
      <item row="6" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>vert sparsity</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="vert_sparsity">
          <property name="decimals">
           <number>5</number>
          </property>
          <property name="singleStep">
           <double>0.000100000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="7" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>vert auxiliary</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="vert_auxiliary">
          <property name="decimals">
           <number>5</number>
          </property>
          <property name="singleStep">
           <double>0.000100000000000</double>
          </property>
          <property name="value">
           <double>0.001000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="8" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>vert increment</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="vert_increment">
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="9" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>vert iteration</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="vert_iteration"/>
        </item>
       </layout>
      </item>
      <item row="10" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>total iteration</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="total_iteration">
          <property name="maximum">
           <number>200</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="11" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>edgeKnn</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="edgeKnn"/>
        </item>
       </layout>
      </item>
      <item row="12" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>edge iteration</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="edge_iteration"/>
        </item>
       </layout>
      </item>
      <item row="13" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_14">
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>edge stepsize</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="edge_stepsize">
          <property name="decimals">
           <number>5</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
